packages/rpc-subscriptions-api/src/slots-updates-notifications.ts

```diff
@@ -1,12 +1,15 @@
-import type { Slot } from '@solana/rpc-types';
+import type { Slot, UnixTimestamp } from '@solana/rpc-types';
 
 type SlotsUpdatesNotificationsApiNotificationBase = Readonly<{
+    /** The newly updated slot */
     slot: Slot;
-    timestamp: bigint;
+    /** The Unix timestamp of the update in milliseconds */
+    timestamp: UnixTimestamp;
```

Comment:
I changed this to use the `UnixTimestamp` type.

Reply:
It might be worth using a different type for timestamps in milliseconds

Eg we use `UnixTimestamp` for block time, which is documented as being in seconds

Reply:
Ahhhh! Good catch. Oops.